src/VoltageMeter.cpp

```diff
@@ -72,6 +91,27 @@ double VoltageMeter::read() {
 }
 
 uint8_t VoltageMeter::readPercentage() {
+  if (!hasReadings()) {
```

Comment:
@Halbelunge  - can you check how we can use your measurements here. Unfortunately I could not find how to map your ADC readings to the ones I get using the slightly calibrated ones. Might be we can base it on the voltage we get?